src/com/tw/go/plugin/nuget/NuGetFeedDocument.java

```diff
@@ -61,7 +61,7 @@ public PackageRevision getPackageRevision(boolean lastVersionKnown) {
         }
         if (getEntries().getLength() > 1)
             throw new NuGetException(String.format("Multiple entries in feed for %s %s", getEntryTitle(), getPackageVersion()));
-        PackageRevision result = new PackageRevision(getPackageLabel(), getPublishedDate(), getAuthor(), getReleaseNotes(), getProjectUrl());
+        PackageRevision result = new PackageRevision(getPackageVersion(), getPublishedDate(), getAuthor(), getReleaseNotes(), getProjectUrl());
```

Comment:
@varchev & @arcaartem - does this make sense? Im asking since I dint write this originally & Im not sure what the author intended.